MIGRATION_GUIDE.md

```diff
@@ -3,6 +3,30 @@
 
 This guide outlines how developers can migrate from older versions of our SDK to newer ones.
 
+## Migrating to v2.4.0
+
+It was recently discovered that the Swift SDK was using legacy event names for some common events,
```

Comment:
Maybe should update the README too unless that was done on the previous PR?

Reply:
I'll check the readme